dotnet-authserver/src/TeacherIdentity.AuthServer/UserClaimHelper.cs

```diff
@@ -82,6 +82,13 @@ public async Task<IReadOnlyCollection<Claim>> GetPublicClaims(Guid userId, Func<
         }
 #pragma warning restore CS0618 // Type or member is obsolete
 
+        var mergedUserIds = await _dbContext.Users.IgnoreQueryFilters()
+            .Where(u => u.MergedWithUserId == userId)
+            .Select(u => u.UserId)
+            .ToListAsync();
+
+        claims.Add(new Claim(CustomClaims.PreviousUserId, String.Join(",", mergedUserIds)));
```

Comment:
Should we not add the claim at all if there are no merged users?

Reply:
I believe you can add multiple claims of the same type so perhaps one per ID (so zero if no merged users)

Reply:
Looks like if you add multiple claims of the same type they are combined into an array